src/ray/common/bundle_spec.h

```diff
@@ -32,7 +34,7 @@ namespace ray {
 /// address and the raylet's port.
 typedef std::function<void()> SpillbackBundleCallback;
 
-const std::string kGroupKeyword = "_group_";
+constexpr const char kGroupKeyword[] = "_group_";
```

Comment:
not a merge blocker, but it's better to use `std::string_view`, because C-style array doesn't provide 
- advanced string manipulation features
- out of bound check